tests/resources/helper_functions.py

```diff
@@ -345,3 +348,67 @@ def transfer_ocean_if_balance_lte(
         )
 
     return ocean_token.balanceOf(recipient) - initial_recipient_balance
+
+
+def get_provider_fees() -> Dict[str, Any]:
+    provider_wallet = get_provider_wallet()
+    web3 = get_web3()
+    provider_fee_amount = 0
+    provider_data = json.dumps({"timeout": 0}, separators=(",", ":"))
+    provider_fee_address = provider_wallet.address
+    provider_fee_token = os.environ.get("PROVIDER_FEE_TOKEN", ZERO_ADDRESS)
+
+    message = Web3.solidityKeccak(
+        ["bytes", "address", "address", "uint256"],
+        [
+            Web3.toHex(Web3.toBytes(text=provider_data)),
+            provider_fee_address,
+            provider_fee_token,
+            provider_fee_amount,
+        ],
+    )
+    signed = web3.eth.sign(provider_fee_address, data=message)
+    signature = split_signature(signed)
+
+    provider_fee = {
+        "providerFeeAddress": provider_fee_address,
+        "providerFeeToken": provider_fee_token,
+        "providerFeeAmount": provider_fee_amount,
+        "providerData": Web3.toHex(Web3.toBytes(text=provider_data)),
+        # make it compatible with last openzepellin https://github.com/OpenZeppelin/openzeppelin-contracts/pull/1622
+        "v": signature.v,
+        "r": signature.r,
+        "s": signature.s,
+    }
+
+    return provider_fee
+
+
+def create_basics(config, web3, data_provider):
```

Comment:
don't we have create_basics in test_ocean_assets.py? If you move it here, please remove it from test_ocean_assets.py.

Reply:
I have moved it to helper functions since I used it in many places.